web/src/components/table/use-cases/generations.tsx

```diff
@@ -763,36 +768,44 @@ export default function GenerationsTable({
         selectedOption={selectedOption}
         setDateRangeAndOption={setDateRangeAndOption}
         actionButtons={
-          <DropdownMenu>
-            <DropdownMenuTrigger asChild>
-              <Button variant="outline" className="ml-auto whitespace-nowrap">
-                <span className="hidden @6xl:inline">
-                  {filterState.length > 0 || searchQuery
-                    ? "Export selection"
-                    : "Export all"}{" "}
-                </span>
-                <span className="@6xl:hidden">Export</span>
-                {isExporting ? (
-                  <Loader className="ml-2 h-4 w-4 animate-spin" />
-                ) : (
-                  <ChevronDownIcon className="ml-2 h-4 w-4" />
-                )}
-              </Button>
-            </DropdownMenuTrigger>
-            <DropdownMenuContent align="end">
-              {Object.entries(exportOptions).map(([key, options]) => (
-                <DropdownMenuItem
-                  key={key}
-                  className="capitalize"
-                  onClick={() =>
-                    void handleExport(key as BatchExportFileFormat)
-                  }
-                >
-                  as {options.label}
-                </DropdownMenuItem>
-              ))}
-            </DropdownMenuContent>
-          </DropdownMenu>
+          plan && isCloudPlan(plan) ? (
```

Comment:
question: this effectively disables the exports for self-hosters right? Is that intended?